lib/render-item.js

```diff
@@ -8,6 +8,7 @@ function renderItem(state, value) {
   const attrValue = valueIsString ? value : value.id;
   const label = valueIsString ? value : value.label;
 
+  item.setAttribute("aria-label", `Choice: ${label}.`);
```

Comment:
Stopped short of changing this as I felt you may have ideas, opinions, insights into the best place to make `Choice: ` localizable.

Begs the question, should there be an optional localization option, so server-side request components forward a locale from the initialization of the component?

It feels mildly out of scope for this, but perhaps having that machinery is more important than serving a particularly rigid feature under the guise of accessibility?

Reply:
Localisation does feel a bit out of scope here ... perhaps we can tackle it in a separate issue, so we can get this improvement over the finish line?